### PR TITLE
Feature/julen/ccw aux testing mt mount fixes

### PR DESCRIPTION
**Pull request by andrewbheyer**

### CODE REVIEW

File: sal_interfaces/MTMount/MTMount_Telemetry.xml [hunk 1]
```diff
@@ -6,291 +6,129 @@
...
      <item>
-          <EFDB_Name>ACW_Status</EFDB_Name>
-          <Description>ACW_Status</Description>
-          <Size_in_bytes>1</Size_in_bytes>
+          <EFDB_Name>CCW_Status</EFDB_Name>
+          <Description>CCW_Status</Description>
+          <IDL_Size>1</IDL_Size>
           <IDL_Type>string</IDL_Type>
-          <Units>unitless</Units>
+          <Units>   </Units>
```
Comment (rbovill) [inline sal_interfaces/MTMount/MTMount_Telemetry.xml]: Units cannot be blank or just whitespace.  Please use `unitless` or `dimensionless`.

File: sal_interfaces/MTMount/MTMount_Telemetry.xml [hunk 2]
```diff
@@ -6,291 +6,129 @@
...
      <item>
-          <EFDB_Name>ACW_Status_Drive_1</EFDB_Name>
-          <Description>ACW_Status_Drive_1</Description>
-          <Size_in_bytes>1</Size_in_bytes>
+          <EFDB_Name>CCW_Status_Drive_1</EFDB_Name>
+          <Description>CCW_Status_Drive_1</Description>
+          <IDL_Size>1</IDL_Size>
           <IDL_Type>string</IDL_Type>
-          <Units>unitless</Units>
+          <Units>   </Units>
```
Comment (rbovill) [inline sal_interfaces/MTMount/MTMount_Telemetry.xml]: Units cannot be blank or just whitespace.  Please use `unitless` or `dimensionless`.

File: sal_interfaces/MTMount/MTMount_Telemetry.xml [hunk 3]
```diff
@@ -6,291 +6,129 @@
...
-          <IDL_Type>double</IDL_Type>
-          <Units>deg</Units>
-          <Count>1</Count>
-      </item>
-     <item>
-          <EFDB_Name>ACW_Positive_Power_Off_switch</EFDB_Name>
-          <Description>ACW_Positive_Power_Off_switch</Description>
-          <IDL_Type>boolean</IDL_Type>
-          <Units>unitless</Units>
+          <Units>   </Units>
```
Comment (rbovill) [inline sal_interfaces/MTMount/MTMount_Telemetry.xml]: Units cannot be blank or just whitespace.  Please use `unitless` or `dimensionless`.

File: sal_interfaces/MTMount/MTMount_Telemetry.xml [hunk 4]
```diff
@@ -6,291 +6,129 @@
...
-          <Units>unitless</Units>
-          <Count>10</Count>
-      </item>
-     <item>
-          <EFDB_Name>timestamp</EFDB_Name>
-          <Description>timestamp</Description>
-          <Size_in_bytes>1</Size_in_bytes>
-          <IDL_Type>double</IDL_Type>
-          <Units>second</Units>
+          <Units>   </Units>
```
Comment (rbovill) [inline sal_interfaces/MTMount/MTMount_Telemetry.xml]: Units cannot be blank or just whitespace.  Please use `unitless` or `dimensionless`.

File: sal_interfaces/MTMount/MTMount_Telemetry.xml [hunk 5]
```diff
@@ -6,291 +6,129 @@
...
      <item>
-          <EFDB_Name>Azimuth_Status</EFDB_Name>
-          <Description>Azimuth_Status</Description>
-          <Size_in_bytes>1</Size_in_bytes>
+          <EFDB_Name>TMA_PXI_Version</EFDB_Name>
+          <Description>TMA_PXI_Version</Description>
+          <IDL_Size>1</IDL_Size>
           <IDL_Type>string</IDL_Type>
-          <Units>unitless</Units>
+          <Units>   </Units>
```
Comment (rbovill) [inline sal_interfaces/MTMount/MTMount_Telemetry.xml]: Units cannot be blank or just whitespace.  Please use `unitless` or `dimensionless`.

File: sal_interfaces/MTMount/MTMount_Telemetry.xml [hunk 6]
```diff
@@ -6,291 +6,129 @@
...
           <Count>1</Count>
       </item>
      <item>
-          <EFDB_Name>Azimuth_Positive_Adjustable_Software_limit</EFDB_Name>
-          <Description>Azimuth_Positive_Adjustable_Software_limit</Description>
+          <EFDB_Name>CCW_Positive_Directional_limit_switch</EFDB_Name>
+          <Description>CCW_Positive_Directional_limit_switch</Description>
           <IDL_Type>boolean</IDL_Type>
-          <Units>unitless</Units>
+          <Units>   </Units>
```
Comment (rbovill) [inline sal_interfaces/MTMount/MTMount_Telemetry.xml]: Units cannot be blank or just whitespace.  Please use `unitless` or `dimensionless`.

File: sal_interfaces/MTMount/MTMount_Telemetry.xml [hunk 7]
```diff
@@ -6,291 +6,129 @@
...
      <item>
-          <EFDB_Name>Azimuth_Interlocks</EFDB_Name>
-          <Description>Azimuth_Interlocks</Description>
-          <Size_in_bytes>1</Size_in_bytes>
+          <EFDB_Name>CCW_Interlocks</EFDB_Name>
+          <Description>CCW_Interlocks</Description>
+          <IDL_Size>1</IDL_Size>
           <IDL_Type>string</IDL_Type>
-          <Units>unitless</Units>
+          <Units>   </Units>
```
Comment (rbovill) [inline sal_interfaces/MTMount/MTMount_Telemetry.xml]: Units cannot be blank or just whitespace.  Please use `unitless` or `dimensionless`.

File: sal_interfaces/MTMount/MTMount_Telemetry.xml [hunk 8]
```diff
@@ -6,291 +6,129 @@
...
           <Count>1</Count>
       </item>
      <item>
-          <EFDB_Name>Azimuth_Negative_Adjustable_Software_limit</EFDB_Name>
-          <Description>Azimuth_Negative_Adjustable_Software_limit</Description>
+          <EFDB_Name>CCW_Negative_Directional_limit_switch</EFDB_Name>
+          <Description>CCW_Negative_Directional_limit_switch</Description>
           <IDL_Type>boolean</IDL_Type>
-          <Units>unitless</Units>
+          <Units>   </Units>
```
Comment (rbovill) [inline sal_interfaces/MTMount/MTMount_Telemetry.xml]: Units cannot be blank or just whitespace.  Please use `unitless` or `dimensionless`.

File: sal_interfaces/MTMount/MTMount_Telemetry.xml [hunk 9]
```diff
@@ -300,5899 +138,98 @@
...
+ <EFDB_Topic>MTMount_Safety_System</EFDB_Topic>
      <item>
-          <EFDB_Name>Azimuth_Current_Drive_10</EFDB_Name>
-          <Description>Azimuth_Current_Drive_10</Description>
-          <IDL_Type>double</IDL_Type>
-          <Units>A</Units>
+          <EFDB_Name>CCW_STO</EFDB_Name>
+          <Description>CCW_STO</Description>
+          <IDL_Type>boolean</IDL_Type>
+          <Units>   </Units>
```
Comment (rbovill) [inline sal_interfaces/MTMount/MTMount_Telemetry.xml]: Units cannot be blank or just whitespace.  Please use `unitless` or `dimensionless`.

File: sal_interfaces/MTMount/MTMount_Telemetry.xml [hunk 10]
```diff
@@ -300,5899 +138,98 @@
...
       </item>
      <item>
-          <EFDB_Name>Azimuth_Current_Drive_11</EFDB_Name>
-          <Description>Azimuth_Current_Drive_11</Description>
-          <IDL_Type>double</IDL_Type>
-          <Units>A</Units>
+          <EFDB_Name>Pull_CordPos</EFDB_Name>
+          <Description>Pull_CordPos</Description>
+          <IDL_Type>boolean</IDL_Type>
+          <Units>   </Units>
```
Comment (rbovill) [inline sal_interfaces/MTMount/MTMount_Telemetry.xml]: Units cannot be blank or just whitespace.  Please use `unitless` or `dimensionless`.

**Comment by andrewbheyer** [comment]
All empty blank spaces changed to be unitless.

File: tests/test_Units.py [hunk 11]
```diff
@@ -46,6 +46,8 @@ def test_units(xmlfile, csc, topic):
     for unit in root.findall(f"./{saltype}/item/Units"):
         if not unit.text:
             assert False, 'Units cannot be blank.'
+        elif " " in unit.text:
+            assert False, 'Units cannot have blank spaces'
```
Comment (rbovill) [inline tests/test_Units.py]: This check is not working properly, as it catches instances of `m / s` which is valid.  Instead I suggest modifying the first condition to be `if not unit.text.replace(" ", ""):` which will allow units formatted like `m / s` but will not allow fields of just spaces.

Comment (andrewbheyer) [reply tests/test_Units.py]: "as it catches instances of m / s which is valid", geat point. Ok I have changed the test now.

**Pull request merged**